lib/matplotlib/tests/test_figure.py

```diff
@@ -66,6 +66,31 @@ def test_align_labels():
     fig.align_labels()
 
 
+@image_comparison(['figure_align_titles'], extensions=['png', 'svg'],
+                  tol=0 if platform.machine() == 'x86_64' else 0.01,
+                  style='mpl20')
+def test_align_titles():
+    fig = plt.figure(layout='tight')
+    gs = gridspec.GridSpec(1, 3)
```

Comment:
Please do not use GridSpec directly.  Users should rarely if ever need to use GridSpec